test/extended/builds/multistage.go

```diff
@@ -23,7 +23,7 @@ var _ = g.Describe("[sig-builds][Feature:Builds] Multi-stage image builds", func
 		testDockerfile = `
 FROM scratch as test
 USER 1001
-FROM centos:7
+FROM registry.access.redhat.com/rhel7
```

Comment:
you're using registry.redhat.io elsewhere, might as well use it here too?

Reply:
update pushed ... for some reason I wasn't convinced the cred on the host was always used, but in hindsight I think you're right

let's see how long it takes to get another clean e2e-cmd run :-(